test/mri/tests/ruby/test_refinement.rb

```diff
@@ -1835,6 +1835,20 @@ def test_used_modules
     assert_equal [ref::RefB, ref::RefA], ref::Combined::USED_MODS
   end
 
+  def test_refined_class
+    refinements = Module.new {
+      refine Integer do
+        int_refinement = self
+      end
+
+      refine String do
+        str_refinement = self
+      end
+    }.refinements
+    assert_equal(Integer, refinements[0].refined_class)
+    assert_equal(String, refinements[1].refined_class)
+  end
+
```

Comment:
**minor:** Not sure whether MRI test should be committed at all.

I suppose there are already existing MRI tests. `#refined_class` is tested in RubySpec ([PR](https://github.com/ruby/spec/pull/1027)) so if you need to add some new test cases - you can find existing specs in `core/refinement/refined_class_spec.rb`.